content/themes/base/index.php

```diff
@@ -7,11 +7,25 @@
 
 get_header();
 
+$folder_path = 'content/themes/base';
+
+if ( file_exists( '/etc/chassis-constants' ) ) {
+	$json_string = file_get_contents( '/etc/chassis-constants' );
+	$data = json_decode( $json_string, true );
+	if ( !empty( $data ) && !empty( $data['synced_folders']['/chassis'] ) ) {
+		$folder_path = $data['synced_folders']['/chassis'].'/content/themes/base';
```

Comment:
We do Nacin spacing in general here so this will be:

```suggestion
		$folder_path = $data['synced_folders']['/chassis'] . '/content/themes/base';
```